src/Components/animation/AnimationEditor.js

```diff
@@ -50,14 +48,43 @@ class AnimationEditor extends React.Component {
   };
 
   onLayout = e => {
+    var cHeight = e.nativeEvent.layout.height;
+    var cWidth = e.nativeEvent.layout.width;
+
     //TODO see why this is needed...
     this.setState({
-      width: e.nativeEvent.layout.width,
-      height: e.nativeEvent.layout.height,
-      dTop: 40,
+      width: cWidth,
+      height: cHeight,
+      dTop: e.nativeEvent.layout.y,
       dLeft: e.nativeEvent.layout.x,
     });
 
+    // Create the elements in the horizontal bar
+    this.initialElements = [];
+
+    var aWidth = cWidth * this.wRatio;
```

Comment:
Same here with a (I guess it is for animation?)

Reply:
Yes it is. I replaced by animation.